system/jlib/jset.hpp

```diff
@@ -155,7 +156,8 @@ extern jlib_decl IBitSet *deserializeThreadSafeBitSet(MemoryBuffer &mb);
  * IOW, e.g. bits 0-sizeof(bits_t) must be set from only 1 thread at a time.
  */
 extern jlib_decl IBitSet *createBitSet(size32_t memSize, const void *mem, bool reset=true);
-// This form allows the size of the bit set to be dynamic. No guarantees about threading.
+// These forms allows the size of the bit set to be dynamic. No guarantees about threading.
+extern jlib_decl IBitSet *createBitSet(unsigned maxBits, bool reset);
```

Comment:
I don't think this function gains you anything.

Reply:
Please see previous comment regard CBitSet.  Shall I remove?